extending-valtimo/tabs/configure-tab-object-type.md

```diff
@@ -0,0 +1,35 @@
+# Setting extra tabs for Zaak object type
+
+On the details page of a case instance, several tabs are shown, including the Zaakobject tab.
+It is possible to configure to display the guides by zaak object type, so the user can see all relevant data in 
+a separate tab.
+
+These tabs can be customized via front-end implementation code.
+
+## Configuring the tabs in the environment file
+
+To customize which tabs are shown, open `environment.ts` in the implementation's `src/environment` folder.
+Here, the `caseObjectTypes` object is defined. It can provide a list of definition names, that contain one or more 
+objects names of types of type `string`, that will be converted into new tabs to be displayed.
+
+Configuration is optional (see example below).
+
+#### **`environment.ts`**
+
+  ```typescript
+  export const environment: ValtimoConfig = { ...
+    caseObjectTypes: {
+      leningen: ['Boom', 'Straatverlichting']
```

Comment:
I would force the documentDefinition name to a string by typing
'leningen': .....

I just noticed an error when I tried to create a caseObjectTypes for generieke-zaak .... that one was only valid when I typed 'generieke-zaak'